### PR TITLE
Main: relative assets url

### PR DESCRIPTION
**Pull request by bpierre**
Related discussion: https://github.com/aragon/aragon-apps/pull/692

### CODE REVIEW

**Comment by bpierre** [comment]
@sohkai I also updated the README (totally forgot to update it for `Main` 😱) and the default directory name for `copy-aragon-ui-assets` is now `aragon-ui`.

This is a breaking change but it shouldn’t affect our apps as I believe we are already always renaming it `aragon-ui` anyway.

File: bin/copy-aragon-ui-assets [hunk 1]
```diff
@@ -51,7 +51,7 @@ async function main(argv, argspec) {
   const destination = path.resolve(
     process.cwd(),
     args._[0],
-    args['--dirname'] || 'aragon-ui-assets'
+    args['--dirname'] || 'aragon-ui'
```
Comment (sohkai) [inline bin/copy-aragon-ui-assets]: I had no idea this wasn't just `aragon-ui` before 😅

File: README.md [hunk 2]
```diff
@@ -17,39 +17,61 @@ Used by:
...
 
+### Quick setup
+
 Install it from npm:
 
 ```sh
 npm install --save @aragon/ui
 ```
 
-Aragon UI require some assets (e.g. fonts) in order to work properly. These need to be copied where they can be accessed by the library, like the `public/` directory of a project using [Create React App](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-assets-outside-of-the-module-system).
```
Comment (sohkai) [inline README.md]: I think it'd still be useful to say why we provide this bit and that the library does requires these assets

File: README.md [hunk 3]
```diff
@@ -17,39 +17,61 @@ Used by:
...
-
-To copy these assets, use the `copy-aragon-ui-assets` command provided by Aragon UI:
+Copy its assets in your public directory:
```
Comment (sohkai) [inline README.md]: in**to**

**Pull request merged**